config/cache.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'default' => env('CACHE_DRIVER', 'file'),
+    'default' => env('CACHE_DRIVER', 'array'),
```

Comment:
Dlaczego? Tzn. Jak chcesz zmienić driver na inny to robisz to w pliku `.env`.
I wydajnościowo wydaje mi się że trzymanie tego w pamięci nie ma sensu. Pliki jednak się lepiej nadają (a serwer i tak mamy z SSD)